webpack/scenes/Subscriptions/SubscriptionReducer.js

```diff
@@ -87,11 +82,7 @@ export default (state = initialState, action) => {
 
     case DELETE_SUBSCRIPTIONS_SUCCESS:
       return state
-        .set('deleteButtonDisabled', true)
-        .set('loading', false);
```

Comment:
There were a lot of actions setting loading to false. I don't see the point especially now that things are using the progress bar. Let me know if I missed something with that